src/main/java/com/mg105/use_cases/Inventory/ItemFactory.java

```diff
@@ -1,14 +1,12 @@
 package com.mg105.use_cases.Inventory;
-
-import com.mg105.entities.HealthPotion;
+import com.mg105.entities.items.*;
 import com.mg105.entities.Item;
-import com.mg105.entities.items.UpgradeToken;
 import com.mg105.utils.ItemConstants;
 
 import java.util.NoSuchElementException;
 
 /**
- * This classes sole responsibility is to create new items
+ * This class' sole responsibility is to create new items
```

Comment:
Apparently the word “class’s” is the singular possessive form of the word “class”,  rather than "class' "